looker studio connection with cloud. (#1437)
ClickHouse/clickhouse-docs: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md

<br/>

Now you can proceed with exploring your data or creating a new report! 

> ## ClickcHouse Cloud Connection

Reviewer: ```suggestion
## Using Looker Studio with ClickHouse Cloud
```

I'd also move this right below "On-premise ClickHouse server setup"

Reviewer: Maybe even above that?

> When using ClickHouse Cloud, you must first file a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL protocol.

Reviewer: Shall we avoid "must" in this case? And protocol -> interface (https://clickhouse.com/docs/en/interfaces/mysql)

```suggestion
When using ClickHouse Cloud, create a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL interface.
```

![Looker Studio Cloud Connection](./images/looker-studio-mysql-cloud.png)

> In the Looker Studio UI, choose the "Enable SSL" option. Upload the [LetsEncryptRoot Cert](https://letsencrypt.org/certificates/) as the server cert. ClickHouse Cloud's SSL certificate is signed by LetsEncrypt. You can download this root cert [here](https://letsencrypt.org/certs/isrgrootx1.pem).

Reviewer: Do we need the first LetsEncryptRoot Cert link? We could just provide a link to download the pem.

Author: Let me reword. We can link the first link for the signed by LetsEncrypt. The first link is to comfort the users who are very sensitive about security. To give context why this cert, etc.

Author: updated.
